Clinical-Genomics/scout
# Cancer SV

This PR adds a functionality - cancer_sv!

Thank you @szilvajuhos! This is Good Stuff that we should try to get into the base repo. We should add some tests and streamline the views a little bit before merging, but you've done a really Good Thing already!

**How to test**:
1. Load a manta cancer sv track!
2. Note that a cancer sv button appears on the case
3. Look and click through functionality on the variants and variant pages for the cancer_sv to see that all is working as far as can be expected from current annotation status.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
